MapleServer2/Types/GameEventUserValue.cs

```diff
@@ -0,0 +1,34 @@
+﻿using MapleServer2.Database;
+using MapleServer2.Enums;
+
+namespace MapleServer2.Types;
+
+public class GameEventUserValue
+{
+    public long Id;
+    public long CharacterId;
+    public GameEventUserValueType EventType;
+    public string EventValue;
```

Comment:
So this value is always a number but in the packet is always a string? If that's the case I would make this a int and only use ToString() on the packet.

Reply:
not always an int. It's sometimes a bool, int, or long